src/shapes/withStylesPropTypes.js

```diff
@@ -0,0 +1,8 @@
+import PropTypes from 'prop-types';
```

Comment:
Why pull this out of react-with-styles?

Reply:
yeah that's actually a good question; can't we import from there instead?

Reply:
`react-with-styles` doesn't actually have this file. :P We fake it in our own internal repo. Probably the right thing to do is to add it to `react-with-styles`

Reply:
aha, yes let's please do that asap :-D